### ISSUE TITLE
Autolink linkToUrl doesn't provide ARIA context for the URL

### ISSUE THREAD

**Issue by jamesmacwhite** [nojimage/twitter-text-php#37]
The function that returns the URL to the tweet on Twitter: https://github.com/nojimage/twitter-text-php/blob/master/lib/Twitter/Text/Autolink.php#L668-L747

Currently doesn't have an `aria-label` which would be beneficial for A11Y, presently the link value is just a URL with the https://t.co path which doesn't describe the link in enough detail.

It would be good for A11Y purposes to set `aria-label` to better describe the link such as "Read more of this tweet on Twitter"

**Comment by nojimage** [comment]
Thank you for your advice.
This is also difficult to change for the same reason as https://github.com/nojimage/twitter-text-php/issues/36#issuecomment-776715917 .

**Comment by jamesmacwhite** [comment]
@nojimage Thanks, I'll look at discussing this upstream.
